CQtDeployer.pro

```diff
@@ -8,8 +8,8 @@
 TEMPLATE = subdirs
 CONFIG += ordered
 
-lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 14) {
-    message(Tests is disabled!)
+lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 12) {
+    warning("Tests are only enabled on Qt 5.12.0 or later version. You are using $$[QT_VERSION].")
     DEFINES += WITHOUT_TESTS
 }
```

Comment:
please add the  
```
include($$PWD/test.pri)
```
in to contains block WITHOUT_TESTS
```
!contains(DEFINES, WITHOUT_TESTS) {
```

Reply:
Well, I did this in another way. Please see https://github.com/QuasarApp/CQtDeployer/compare/cae9981e5bf62ccd13c1110ff3ddfc5a614448cf...ff9690b1155d40355a3adbe433cc9cf080298e5b .